### ISSUE TITLE
1 is composite according to the new assumptions

### ISSUE THREAD

**Issue by pelegm** [sympy/sympy#8808]
Following #8807, both in master and in `asmeurer/newassump`, `ask(Q.composite(1))` is `True`.

I guess the problem is in that line:
```python
    Equivalent(Q.prime, Q.integer & Q.positive & ~Q.composite),
```
according to which, since `1` is not prime, but is a positive integer, it must not be composite.

I guess the only way to overcome this is to implement relationals, and replace `Q.positive` with something that will signifies `>1`.

**Comment by aka001** [comment]
I want to solve this issue. Can you please help me in getting started?
Is there any ask key defined from "one"? 
If there's no ask-key defined for "one", do we need to define a key for "one" to solve this issue?

**Comment by pelegm** [comment]
> Can you please help me in getting started?

I don't think so. I think that `ask` should be implemented on relationals first, and I don't see that coming soon...

**Comment by pelegm** [comment]
Perhaps we can fix it similarly to #8809: replace `Equivalent` with `Implies`. But that's only a partial solution, since we still need relationals for the other direction, I guess.

**Comment by aktech** [comment]
Fixed here #8860 
Should be closed.

**Comment by debugger22** [comment]
Fixed in https://github.com/sympy/sympy/pull/8860
